spread.yaml

```diff
@@ -280,6 +298,64 @@ backends:
             - ubuntu-18.10-arm64:
                 username: ubuntu
                 password: ubuntu
+            # Disco
```

Comment:
Should probably read: `# Disco Dingo`

Reply:
So far all of the entries only have the codename, so this is probably fine